app/lib/widgets/CustomChatInput.dart

```diff
@@ -409,7 +409,7 @@ class AttachmentWidget extends StatelessWidget {
             _roomController.handleMultipleImageSelection(context, roomName);
             break;
           case 'gif':
-            //gif handle
+            _roomController.handleImageSelection(context);
```

Comment:
Would it be okay to redirect user to image picker for this ? Or put ```not implemented feature yet```  widget?

Reply:
This PR has no GIF animation preview stage.
If animation is needed, I will fix later.